api/ruvnl-morning.ts

```diff
@@ -1,9 +1,25 @@
+import * as fs from 'fs';
 import type { VercelRequest, VercelResponse } from '@vercel/node';
 import { CreateEmailResponse, Resend } from "resend";
 
+const logApiEndpoint = (endpoint: string) => {
+  const timestamp = new Date().toISOString();
+  const logMessage = `${timestamp} - API endpoint hit: ${endpoint}\n`;
+  
+  fs.appendFile('logs.txt', logMessage, (err) => {
```

Comment:
I would remove this saving the logs to a file, as we are running this on Vercel (serverless)

Reply:
> I would remove this saving the logs to a file, as we are running this on Vercel (serverless)

where should this be logged then

Reply:
see other logging messages like [this](https://github.com/openclimatefix/quartz-mailer/blob/main/api/ruvnl-morning.ts#L125). it basically gets logged to the console, and we can log on to Vercel and see what happened

Reply:
Ah, because this is a serverless environment, we have no log persistence – unless you mean for this feature to just be for realtime debugging? Sorry wasn't aware of this issue/task at all, to me this would be a Sentry or similar thing really